content/docs/concepts/security/index.md

```diff
@@ -771,6 +770,51 @@ spec:
     name: "products-viewer"
 {{< /text >}}
 
+### Using Istio authorization on plain TCP protocol
+
+The above examples showed a typical way to use Istio authorization on services using HTTP protocol.
+In that case, all fields in a `ServiceRole` and `ServiceRoleBinding` are supported.
+
+Istio authorization also supports services using plain TCP protocol, like mongodb. In this case, you
+specify the `ServiceRole` and `ServiceRoleBinding` the same way as for a HTTP service, except that
+you cannot use the fields that are only applicable to HTTP services, which includes `paths` and `methods`
+in `ServiceRole`, and `group` in `ServiceRoleBinding`.
+
+For the supported constraints and properties that could be used for TCP service, please refer to
+[constraints and properties page](/docs/reference/config/authorization/constraints-and-properties/).
+
+If any HTTP only fields are used for a TCP service, the whole `ServiceRole` or `ServiceRoleBinding`
```

Comment:
This doesn't quite parse well.  Could you rework the sentence our tell me what you mean here, and I can help with that?  I think what you want to sa is "the `ServiceRole` or `ServiceRoleBinding` custom resource will be ...